.github/workflows/lint-ts.yml

```
@@ -0,0 +1,33 @@
+
+      - name: 🏃‍♂️ Run Biome PR
+        if: github.event_name == 'pull_request'
+        uses: mongolyy/reviewdog-action-biome@v1
+        with:
+          github_token: ${{ secrets.GITHUB_TOKEN }}
+          reporter: github-pr-review
+          fail_on_error: true
+
+      - name: 🏃‍♂️ Run Biome PR
+        if: github.event_name != 'pull_request'
+        run: bun run lint
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)